app/controllers/admin_controller.rb

```diff
@@ -7,7 +7,7 @@ def assets; end
   def promote_admin
     @user = User.find params[:id]
     unless @user.nil?
-      if logged_in_as(['admin'])
+      if logged_in_as(%w(admin))
```

Comment:
This is in the rubocop config, but I'm not sure about it -- actually `['admin']` seems more readable and common of a usage... what do you think? Could we adapt the config to use this simpler format? Thank you!!!!